### PR TITLE
Migrate Layout to Tailwind

### PR DESCRIPTION
**Pull request by oreqizer**
Needed to work with Grid.

 Storybook: https://orbit-mainframev-oreqizer-fix-layout.surge.sh

### CODE REVIEW

**Review by mainframev — APPROVED** [review]

**Review by mainframev — CHANGES_REQUESTED** [review]
it's visually different than in master (width)

File: packages/orbit-components/src/Layout/index.tsx [hunk 1]
```diff
@@ -18,30 +15,20 @@ const getChildrenProps = (type: Type, key: string) => {
...
-StyledLayout.defaultProps = {
-  theme: defaultTheme,
-};
-
 const Layout = ({ children, type, dataTest }: Props) => (
-  <StyledLayout {...LAYOUT_SETTINGS[type]} dataTest={dataTest}>
+  <Grid
+    {...LAYOUT_SETTINGS[type]}
+    width="100%"
+    className="px-md de:p-lg mx-auto my-0 box-border py-0"
```
Comment (mainframev) [inline packages/orbit-components/src/Layout/index.tsx:22]: StyledLayout before had "width: 100%", not it does not have this class, could you add? That should fix it

Comment (oreqizer) [reply packages/orbit-components/src/Layout/index.tsx]: tried that, didn't work

Comment (mainframev) [reply packages/orbit-components/src/Layout/index.tsx]: found the [old build from may](https://9ca42447.orbit.pages.dev/components/layout/layout/) (where it's supposed to be ok), so I guess it's correct, that on tablet and above it does not have full width

**Review by mainframev — APPROVED** [review]
LGTM

File: packages/orbit-components/src/utils/Grid/types.d.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ export interface BasicProps extends Common.SpaceAfter {
 
 export interface Props extends Common.Globals, Common.SpaceAfter, BasicProps {
   readonly as?: string;
+  readonly className?: string;
```
Comment (mainframev) [inline packages/orbit-components/src/utils/Grid/types.d.ts:21]: separeted feat commit 🙏🏻 and do not forget also to add that to flow file ;)

Comment (oreqizer) [reply packages/orbit-components/src/utils/Grid/types.d.ts:21]: done 👌

**Pull request merged**